app/views/topics/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2>Topics</h2>
```

Comment:
Generally the top heading on a page (or section) should always be a `<h1/>`. It's more correct from a semantic point of view and may (depending on who you ask) impact SEO (https://www.hobo-web.co.uk/headers/). 

If you want headings to appear in different sizes, this should be done in CSS.